### ISSUE TITLE
MCDU's GPS Monitor Ground Speed (GS) is incorrect

### ISSUE THREAD

**Issue by Prescia** [flybywiresim/a32nx#5827]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-09-13T21:13:14+00:00",
  "ref": "master",
  "sha": "6dc2a169d5b856401327707f033191e7e62c82e0",
  "actor": "beheh",
  "event_name": "manual"
}
```

### Describe the bug

On the MCDU > Data > GPS Monitor, the Ground Speed (GS) is incorrect, highly under the correct value. I am attaching a screenshot that displays the issue.

According to the Aircraft checklist for Inconsistent IAS (which in-game can be caused by the pilot forgetting to turn on the pitot tube heater, or using RandFailures and having a Pitot Tube blockage or Static Port Blockage), among the troubleshoot steps is to use said Ground Speed for reference.

I found the issue exactly because of that, I had a Inconsistent IAS and since the A320 doesn't have VOR lock to check your speed, I researched the procedure and it suggested the GPS monitor page, and then found out the GS is incorrect. 

![MCDU-DATA-GPS](https://user-images.githubusercontent.com/8838849/133515429-f3d8568b-ec82-416f-8907-d270d697a003.jpg)

The GPS GS should also be 474

### Expected behavior

GPS Monitor's GS should be the same as displayed on the ND.

During a Pitot Blockage or IAS issue, the GS on the ND will be zero, but the GPS GS should remain working.

### Steps to reproduce

1. Access the MCDU > Data > GPS Monitor page
2. Notice GS is wrong (always extremely lower)

### References (optional)

Found the procedure to use the GPS Monitor for speed on the A320 checklist for unreliable IAS

### Discord Username (optional)

Caiobrz#9021

**Comment by donstim** [comment]
Thanks for posting this issue. Problem was that the GPS ground speed on the MCDU Monitor GPS page was in meters/second. Should be fixed in PR #5848

**Comment by donstim** [comment]
> Thanks for the fix. Just realized it is still wrong in the IRS page (Data > IRS > IRS 1|2|3)

Okay, thanks for identifying this one as well. I'm on it.
